RoomsEyeView/Logic/CMakeLists.txt

```diff
@@ -10,6 +10,7 @@ set(${KIT}_INCLUDE_DIRECTORIES
   ${vtkSlicerBeamsModuleMRML_INCLUDE_DIRS}
   ${vtkSlicerBeamsModuleLogic_INCLUDE_DIRS}
   ${RapidJSON_INCLUDE_DIR}
+  ${vtkIECTransformLogic_INCLUDE_DIRS}

```

Comment:
In the Beams module logic CMake there was this line
`find_package(vtkIECTransformLogic REQUIRED)`
Why is it not needed here? Is it needed in Beams logic?

Reply:
I've done it now in all files, as done with plastimatch find_package statements.